tests/DbaaS/api/dbaasAPI.js

```diff
@@ -31,6 +31,19 @@ module.exports = {
     );
   },
 
+  async apiUnregisterAllCluster() {
+    const body = {};
```

Comment:
I think there is no need to create an empty object here, just pass `{}` to the request itself

Reply:
thanks will update.